src/Components/QuestionListSearch/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { Question } from "../../Apis/types";
+
+interface Props {
+    questions: Question[];
+    searchTerm: string;
+}
+
+export const FilterBySearchTerm = ({ questions, searchTerm }: Props) => {
+    let questionsFiltered = questions;
+
+    return questionsFiltered.filter(({ question }) => {
+        question = question.toLocaleUpperCase();
+        searchTerm = searchTerm.toLocaleUpperCase();
```

Comment:
this probably should have been done at the top of the order so we know exactly where it's happening or even in the props feild.